src/bridge/MatrixHandler.ts

```diff
@@ -97,7 +117,7 @@ export class MatrixHandler {
 
     constructor(
         private ircBridge: IrcBridge,
-        config: {eventCacheSize?: number} = {},
+        private config: Config = {},
```

Comment:
This config won't be hot-reloaded, might be worth doing the do now that the config is more useful. See `IrcHandler` / `IrcBridge` on how we do config reloads.

Reply:
It wasn't actually loaded at all /o\ I fixed that and IrcHandler config (which was broken for similar reasons) in 48c6e48f.